packages/amplication-client/src/Resource/git/GitActions/ExistingConnectionsMenu.scss

```
@@ -33,6 +33,7 @@
         overflow: hidden;
         white-space: nowrap;
         text-overflow: ellipsis;
+        color: var(--menu-item-active-color)
```

```suggestion
        color: var(--menu-item-active-color);
```
There is a typo, missing `;`. Please update it :)

---

Thanks @overbit for your warning ! It's updated now